app/src/main/java/com/mikhailovskii/weatherandroid/data/firebase/FirebaseModel.kt

```diff
@@ -0,0 +1,141 @@
+package com.mikhailovskii.weatherandroid.data.firebase
+
+import com.google.firebase.firestore.FirebaseFirestore
+import com.mikhailovskii.weatherandroid.data.entities.StickerPack
+import com.mikhailovskii.weatherandroid.data.entities.User
+import com.twitter.sdk.android.core.Result
+import timber.log.Timber
+
+class FirebaseModel {
+
+    private val db = FirebaseFirestore.getInstance()
+
+    fun getStickersCollectionList(callback: FirebaseDataCallback<ArrayList<StickerPack>>?) {
+        db.collection("stickers").get().addOnSuccessListener { result ->
+            val stickerPackList = ArrayList<StickerPack>()
+
+            result.forEach { document ->
+                val stickerPack = document.toObject(StickerPack::class.java)
+                stickerPackList.add(stickerPack)
+            }
```

Comment:
@SergeiMikhailovskii what do you think of a one line statement?
```kotlin
result.forEach { stickerPackList.add(it.toObject(StickerPack::class.java)) }
```

Reply:
you may also use `map` if the `result` type supports it. Then it would look like the following
```kotlin
val stickerPackList = result.map { it.toObject(StickerPack::class.java) }
```